packages/bootstrap/scss/_variables.scss

```diff
@@ -339,6 +339,9 @@ $popup-shadow: 0 2px 4px 0 rgba(0, 0, 0, .03), 0 4px 5px 0 rgba(0, 0, 0, .04) !d
 $grouping-header-text: $bg-color !default;
 $grouping-header-bg: $text-color !default;
 $grouping-border: $border-color !default;
+$frozen-grouping-header-border-width: 2px !default;
+$frozen-grouping-header-color: #6c757d !default;
```

Comment:
check if we have similar or the same color as a variable for the bootstrap theme